runtime/scripts/question.js

```diff
@@ -1195,8 +1205,10 @@ Question.prototype = /** @lends Numbas.Question.prototype */
                 credit = marks>0 ? score/marks : 0;
                 break;
         }
+        
         this.score = score;
         this.marks = marks;
+        this.events.trigger('calculateScore', score, marks);
```

Comment:
you don't need to give `score` and `marks` as arguments, because they're properties of the question object